app/src/main/java/ca/llamabagel/transpo/ui/search/SearchViewModel.kt

```diff
@@ -7,22 +7,29 @@ package ca.llamabagel.transpo.ui.search
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import ca.llamabagel.transpo.data.SearchRepository
+import ca.llamabagel.transpo.ui.search.viewholders.BaseAdapterItem
+import kotlinx.coroutines.launch
 import javax.inject.Inject
 
 enum class KeyboardState {
     OPEN, CLOSED
 }
 
-class SearchViewModel @Inject constructor() : ViewModel() {
+class SearchViewModel @Inject constructor(private val searchRepository: SearchRepository) : ViewModel() {
 
-    private val _keyboardState = MutableLiveData<KeyboardState>()
+    private val _keyboardState = MutableLiveData<KeyboardState>().apply { value = KeyboardState.OPEN }
     val keyboardState = _keyboardState as LiveData<KeyboardState>
```

Comment:
```suggestion
    val keyboardState: LiveData<KeyboardState> = _keyboardState
```

Reply:
Is this better than casting it directly? Or just an alternative?

Reply:
Syntactically it just makes more sense. It's just an implicit cast otherwise.